src/pages/news/index.tsx

```diff
@@ -1,6 +1,89 @@
+      title: 'Visit Of The Training Center',
+      description:
+        'Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ' +
+        'ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame',
+      imageURL:
+        'https://www.wallpapers13.com/wp-content/uploads/2022/12/Snowy-christmas-840x525.jpg',
+      date: '22.01.2022',
+    },
+    {
+      id: uuid(),
+      title: 'Visit Of The Training Center',
+      description:
+        "Lorem ipsum dolor sit ame ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame'",
+      imageURL:
+        'https://www.wallpapers13.com/wp-content/uploads/2022/12/Snowy-christmas-840x525.jpg',
+      date: '22.01.2022',
+    },
+    {
+      id: uuid(),
+      title: 'Visit Of The Training Center',
+      description:
+        "Lorem ipsum dolor sit ame ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame'",
+      imageURL:
+        'https://www.wallpapers13.com/wp-content/uploads/2021/05/Grand-Canyon-National-Park-Arizona-USA-Red-Clouds-Sunset-Landscape-840x525.jpg',
+      date: '22.01.2022',
+    },
+    {
+      id: uuid(),
+      title: 'Visit Of The Training Center',
+      description: 'Lorem ipsum dolor sit ame',
+      imageURL:
+        'https://www.wallpapers13.com/wp-content/uploads/2021/04/Milky-Way-over-Crater-Lake-National-Park-Oregon-United-States-840x525.jpg',
+      date: '22.01.2022',
+    },
+    {
+      id: uuid(),
+      title: 'Visit Of The Training Center',
+      description:
+        "Lorem ipsum dolor sit ame ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame'",
+      imageURL:
+        'https://www.wallpapers13.com/wp-content/uploads/2021/04/Milky-Way-over-Crater-Lake-National-Park-Oregon-United-States-840x525.jpg',
+      date: '22.01.2022',
+    },
+    {
+      id: uuid(),
+      title: 'Visit Of The Training Center',
+      description:
+        "Lorem ipsum dolor sit ame ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame Lorem ipsum dolor sit ame'",
+      imageURL:
+        'https://www.wallpapers13.com/wp-content/uploads/2021/04/Milky-Way-over-Crater-Lake-National-Park-Oregon-United-States-840x525.jpg',
+      date: '22.01.2022',
+    },
+    {
+      id: uuid(),
+      title: 'Visit Of The Training Center',
+      description: 'Lorem ipsum dolor sit ame',
+      imageURL:
+        'https://www.wallpapers13.com/wp-content/uploads/2021/04/Milky-Way-over-Crater-Lake-National-Park-Oregon-United-States-840x525.jpg',
+      date: '22.01.2022',
+    },
+  ]
 
-export default News
+  return (
+    <Container>
+      <PageHeader
+        title="News"
+        paths={['Home', 'News']}
+      />
+      <p className="text-base text-gray-500">The Latest News From our Foundation</p>
+      {newsList.map((news: INewsProps) => (
+        <SingleNews news={news} />
```

Comment:
For now leve as it is, but we need to keep in mind that we need only ID here not more data

Reply:
Ok